src/schemaGenerator.ts

```diff
@@ -370,6 +370,20 @@ function addResolveFunctionsToSchema(
       );
     }
 
+    // Check that there's a resolver for this schema prop
+    // sometimes it can be an object or a function, so check
+    // that it is not falsey AND is iterable
+    console.log('typeName', typeName, resolveFunctions);
```

Comment:
This `console.log` got left behind...

Reply:
Good catch! Thanks @kbrandwijk appreciate your quick response :)